openassessment/templates/openassessmentblock/staff_area/oa_staff_assessment.html

```diff
@@ -0,0 +1,51 @@
+{% load i18n %}
+{% spaceless %}
+
+    {% block body %}
+    <div class="ui-toggle-visibility__content">
+        <div class="wrapper--staff-assessment">
+            <div class="step__instruction">
+                <p>{% trans "Allows you to override the current learner's grade using the problem's rubric." %}</p>
```

Comment:
@andy-armstrong Can we change this to "Override this learner's current grade using the problem's rubric."